metaci/conftest.py

```diff
@@ -1,51 +1,107 @@
 import factory
 import factory.fuzzy
 
+import numbers
+import random
+
 from metaci.plan.models import Plan, PlanRepository
 from metaci.testresults.models import TestResult, TestMethod, TestClass
-from metaci.build.models import BuildFlow, Build
+from metaci.build.models import BuildFlow, Build, BUILD_STATUSES, BUILD_FLOW_STATUSES
 from metaci.repository.models import Branch, Repository
 
 from metaci.users.models import User
 
+BUILD_STATUS_NAMES = (name for (name, label) in BUILD_STATUSES)
+BUILD_FLOW_STATUS_NAMES = (name for (name, label) in BUILD_FLOW_STATUSES)
+
+
+def do_logs():
```

Comment:
This looks unused?

Reply:
I should add a comment to this effect:

"""Call this function at the module level to get more insight into what FactoryBoy is dong under the covers.""" 

Some future maintainer might need it and not know where to find it in the FactoryBoy docs.

Reply:
https://github.com/SFDO-Tooling/MetaCI/pull/408/commits/cb9ac02e086d411c208d84d35e2119e5e8efb7bd#r275034674